charts/spinnaker/templates/configmap/halyard-config.yaml

```
@@ -133,6 +133,10 @@ data:
                 {{ if $account.readPermissions }}--read-permissions {{ join " --read-permissions " $account.readPermissions }}{{ end }} \
                 {{ if $account.writePermissions }}--write-permissions {{ join " --write-permissions " $account.writePermissions }}{{ end }} \
                 --provider-version v2
+    {{- range $index, $customResource := .Values.kubeConfig.customResources }}
```

```suggestion
    {{- range $index, $customResource := $.Values.kubeConfig.customResources }}
```

Since it's being evaluated in the scope of a `range` loop (so the context changes).